charts/dbildungs-iam/Chart.yaml

```diff
@@ -1,12 +1,6 @@
 apiVersion: v2
 name: dbildungs-iam
 version: 0.2.0
-
 description: dBildungs-IAM
 type: application
-
-dependencies:
-  # In non-test-deployments we expect keycloak to be provided for us thus it needs to be enabled
-  - name: dbildungs-iam-keycloak-dev
-    version: "*"
-    condition: keycloakEnabled
\ No newline at end of file
+AppVersion: ""
```

Comment:
I think this should be `appVersion` (despite being referenced as `.Chart.AppVersion`). As long as we keep the dbildungs-iam-keycloak-dev subchart (can be removed if the automatic keycloak deployment works), we should probably keep it as dependency here, too.